src/api/server/configuration.rst

```diff
@@ -16,316 +16,14 @@
 Configuration
 =============
 
-The CouchDB Server Configuration API provide an interface to query and update
-the various configuration values within a running CouchDB instance.
+The classical CouchDB Server Configuration API located at ``/_config`` will
+return a 404 in CouchDB 2.0 and got replaced by :ref:`HTTP API <api/node>`.
 
-``/_config``
```

Comment:
I agree with @janl: we should not document the backdoor interface as we discourage people to use it, except for endpoints that we want that people use them (e.g. compaction)

Btw: is there any reason to keep the routes we don't want people to use on the backdoor ports, e.g. /{db}/_all_docs?